.editorconfig

```diff
@@ -0,0 +1,26 @@
+root = true
+
+[*]
+trim_trailing_whitespace = true
+end_of_line = lf
+
+[*.py]
+indent_style = space
+indent_size = 4
+charset = utf-8
+insert_final_newline = true
+max_line_length = 120
```

Comment:
We have a max line length of 80 characters for Python (per the [Google Python style guide](http://google.github.io/styleguide/pyguide.html#32-line-length)).

Reply:
changed